go-runtime/ftl/ftltest/ftltest.go

```diff
@@ -30,23 +30,27 @@ type OptionsState struct {
 	secrets                 map[string][]byte
 	databases               map[string]modulecontext.Database
 	mockVerbs               map[schema.RefKey]modulecontext.Verb
+	mockMaps                map[schema.RefKey]modulecontext.Map
 	allowDirectVerbBehavior bool
+	allowDirectMapBehavior  bool
 }
 
 type Option func(context.Context, *OptionsState) error
 
 // Context suitable for use in testing FTL verbs with provided options
 func Context(options ...Option) context.Context {
-	ctx := log.ContextWithNewDefaultLogger(context.Background())
-	ctx = internal.WithContext(ctx, newFakeFTL())
-	name := reflection.Module()
-
 	state := &OptionsState{
 		configs:   make(map[string][]byte),
 		secrets:   make(map[string][]byte),
 		databases: make(map[string]modulecontext.Database),
 		mockVerbs: make(map[schema.RefKey]modulecontext.Verb),
+		mockMaps:  make(map[schema.RefKey]modulecontext.Map),
```

Comment:
I think it would be preferable to encapsulate this in the fake and add a `addMapMock()` method to it, rather than expose the internal data structure here.

Reply:
done!